TESTING.md

```diff
@@ -14,11 +14,55 @@ In this article you will:
   - Generate C# projects corresponding to all the templates in various configurations (no auth, single-org, single-org calling graph, single-org calling web API, Individual B2C, Individual B2C calling web API (for the web API and the Blazorwasm hosted templates, as B2C does not support OBO)).
   - Configure the projects with existing Azure AD and B2C apps and client secrets. This is done by a configuration file named `configuration.json`. You will need to add the client secrets (see below).
   - Build the generated projects (which are grouped in a solution named `test.sln`).
-- Manually test (for now) the generated projects.
+- [Manually test (for now) the generated projects](How-to-test-the-configured-projects-manually).
 
-## How to generate the test projects
+## How to generate the test projects from a NuGet Package
```

Comment:
We might want to add a sentence saying that it's possible to generate test projects from 2 sources:
- the repo (link to the section below)
- a NuGet package created from a release build.

?